test/e2e/specs/editor/plugins/iframed-block.spec.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Iframed block', () => {
+	test.beforeEach( async ( { requestUtils, admin } ) => {
+		await requestUtils.activatePlugin( 'gutenberg-test-iframed-block' );
+		await admin.createNewPost( { postType: 'page' } );
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin( 'gutenberg-test-iframed-block' );
+	} );
+
+	test( 'Should save the changes', async ( { editor, page } ) => {
+		await editor.insertBlock( { name: 'test/iframed-block' } );
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+
+		await expect(
+			page.locator( 'role=document[name="Block: Iframed Block"i]' )
+		).toContainText( 'Iframed Block (set with jQuery)' );
+
+		// open page from sidebar settings
+		await page.locator( 'role=button[name= "Page"i]' ).first().click();
```

Comment:
We can target it using `>>` to increase confident without having to use `.first()`.

```suggestion
		await page.click( 'role=region[name="Editor settings"i] >> role=button[name="Page"i]' );
```